recipes/secretstorage/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "SecretStorage" %}
+{% set version = "3.0.1" %}
+{% set sha256 = "819087ca89c0d6c5711692f41fb26f786af9dcc5bb89d567722a66edfbb2a689" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python >=3.5
+    - pip
+    - setuptools
```

Comment:
Do we need both pip and setuptools?

Reply:
I think it was supposed to be the other way around, i.e. remove setuptools and keep pip :)